client/src/pages/Location.js

```diff
@@ -26,6 +28,7 @@ const LocationDiv = styled.div`
 
 const Location = () => {
   const { name } = useParams();
+  const { toggleFavorite, favorites } = useFavorites("favorites", []);
   const { isLoading, error, data: LocationByName } = useQuery(
```

Comment:
Not related to this pull request, but it's weird that the naming convention changed.
```suggestion
  const { isLoading, error, data: locationByName } = useQuery(
```